### PR TITLE
no time interp flag

### PR DESCRIPTION
**Pull request by rkumar-slim**
added Flag to check whether we need time interpolation or not

### CODE REVIEW

File: src/TimeModeling/time_modeling_serial.jl [hunk 1]
```diff
@@ -36,24 +36,42 @@ function time_modeling(model_full::Model, srcGeometry::Geometry, srcData, recGeo
 
     # Extrapolate input data to computational grid
     if mode==1  # F and J
+        if dtComp==srcGeometry.dt[1]
+        qIn = srcData[1];
+        else
         qIn = time_resample(srcData[1],srcGeometry,dtComp)[1]
+        end
```
Comment (mloubout) [inline src/TimeModeling/time_modeling_serial.jl]: So from what I see this is the same if/else everywhere. So i would leave this function as it is and add the if/else inside the `time_resample` resample instead.

Comment (rkumar-slim) [reply src/TimeModeling/time_modeling_serial.jl]: ok, I will change the time-resample and keep this clean

Comment (philippwitte) [reply src/TimeModeling/time_modeling_serial.jl]: Thanks

Comment (rkumar-slim) [reply src/TimeModeling/time_modeling_serial.jl]: @mloubout  when u say odd things, you mean its not indenting the loops properly?

**Comment by mloubout** [comment]
No i mean there is a bunch of empty/non-empty lines showing everywhere in the diff for some reasons.

For the indentation:
```
if ...
    stuff
end
```
with 4 space

**Comment by rkumar-slim** [comment]
I added the if/else statement in time_resample and removed the changed from time_modleing_serial.jl

**Comment by mloubout** [comment]
Please remove binaries (*.pyc) this ones should never be commited.

**Comment by philippwitte** [comment]
Looks good to me now

**Comment by rkumar-slim** [comment]
is there anything else from my side?

**Comment by mloubout** [comment]
GTG to me. @rkumar you all done

**Pull request merged**